src/main/java/edu/bu/metcs673/trackr/common/CommonConstants.java

```diff
@@ -8,6 +8,17 @@
  */
 public class CommonConstants {
 
+	// Allowed patterns in JWT filter (security related)
+	public static final String AUTHORIZATION_HEADER = "Authorization";
+	public static final String BEARER_PREFIX = "Bearer ";
+	public static final String FAVICON_PATH = "/favicon.ico";
+	public static final String IMAGES_PATH = "/images/.*";
+	public static final String GENERATED_ASSETS_PATH = "/built/.*";
+	public static final String ROOT_PATH = "/";
+	public static final String REGISTER_PATH = "/register";
+	public static final String LOGIN_PATH = "/login";
+	public static final String LOGOUT_PATH = "/logout";
```

Comment:
Those are not really common but it's a difference in approaches. I like to keep constants related to a class in that class to avoid creating a lot of dependencies on a single file. This is perfectly fine.